index.html

```diff
@@ -5,11 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- [ ] Please the height of the navigation doesn't match the design 
![image](https://github.com/norbiboy1/Portfolio/assets/74744787/16174803-a588-4c1b-b10a-9f6895f487fa)
![image](https://github.com/norbiboy1/Portfolio/assets/74744787/0ce66a66-1a7c-46cc-9979-4b408ae1f247)